gpio_controllers/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>gpio_controllers</name>
+  <version>1.0.0</version>
```

Comment:
Should the version number not be 4.12.1 as the rest of the repository ?

Reply:
Done